components/QuestionForm/QuestionForm.tsx

```diff
@@ -62,6 +72,23 @@ function QuestionFormInner({
   onSuccess?: () => void;
 }) {
   const upsertQuestion = useUpsertQuestionMutation();
+  const { data, isLoading } = useIssueTagsQuery();
+
+  const labelOptions = data?.allIssueTags;
+  const [selectedLabels, setSelectedLabels] = useState<
+    {
+      id: string;
+      label: string;
+    }[]
+  >([]);
```

Comment:
We will want to populate this state using existing question issue tags if they exist as this form is also used to edit existing questions

Reply:
Seeing your note about this—not a blocker